components/NavBar/NavBar.module.css

```
@@ -46,11 +46,23 @@
 .searchPlaceholder {
   width: 268px;
   height: 36px;
-  border-radius: 4px;
-  border: 1px solid #8e8e8e;
+  border-radius: 8px;
   background: none;
   display: flex;
   align-items: center;
   color: #8e8e8e;
   gap: 2px;
+  background-color: #efefef;
+  border: 0;
+}
+.inputSection {
+  display: flex;
+  align-content: center;
+  justify-content: center;
+}
+.cancelIcon {
+  color: rgba(38, 38, 38, 0.3);
```

That's the color there.
```suggestion
  color: rgb(199, 199, 199);
```